src/Stryker.Core/Stryker.Core/Initialisation/InitialisationProcess.cs

```diff
@@ -120,7 +120,7 @@ private void InitializeDashboardReporter(DashboardReporter dashboardReporter, Pr
 
                 if (!projectInfo.ProjectUnderTestAnalyzerResult.Properties.TryGetValue("TargetPath", out var targetPath))
                 {
-                    throw new StrykerInputException($"Can't read {subject.ToLowerInvariant()} because the TargetPath property was not found in {projectFilePath}");
```

Comment:
Sorry, I messed my branches, cherry picked the commit and did not even verify if the code was still compiling. 😐

Reply:
I fixed it for you :)